ipaclient/plugins/dns.py

```diff
@@ -417,6 +418,63 @@ def interactive_prompt_callback(self, kw):
 
 @register(override=True, no_fail=True)
 class dns_update_system_records(MethodOverride):
+    record_groups = ('ipa_records', 'location_records')
+
+    def get_options(self):
+        for option in super(dns_update_system_records, self).get_options():
+            yield option
+        yield Str(
+            'out?',
+            include='cli',
+            doc=_('file to store DNS records in nsupdate format')
+        )
```

Comment:
Not that this won't work, but you can use `takes_options` as usual here, `MethodOverride` will tak care of properly merging the remote and local options.